test/integration/spec/rtcpeerconnection.js

```diff
@@ -920,9 +920,14 @@ function testClose(signalingState) {
       ];
```

Comment:
@makarandp0 Can we do something like this:
```js
var events = isChrome && browserVersion >= 80 ? ['signalingstatechange'] : [
  'iceconnectionstatechange',
  'signalingstatechange'
];
```
? Then, we don't need the `if` statement in `events.forEach`.